src/taxonomy/taxonomy-menu/TaxonomyMenu.test.jsx

```diff
@@ -86,20 +92,24 @@ describe.each([true, false])('<TaxonomyMenu iconMenu=%s />', async (iconMenu) =>
     jest.clearAllMocks();
   });
 
-  test('should open and close menu on button click', () => {
+  test('should open and close menu on button click', async () => {
     const { getByTestId, queryByTestId } = render(<TaxonomyMenuComponent iconMenu={iconMenu} />);
 
     // Menu closed/doesn't exist yet
     expect(queryByTestId('taxonomy-menu')).not.toBeInTheDocument();
 
     // Click on the menu button to open
-    fireEvent.click(getByTestId('taxonomy-menu-button'));
+    await act(async () => {
+      fireEvent.click(getByTestId('taxonomy-menu-button'));
+    });
```

Comment:
We don't need to wrap the `fireEvent` with `act`.
If you are having errors due to async code, you could use the code below with `findByTestId` instead of `getByTestId`:
```suggestion
    expect(await findByTestId('taxonomy-menu-button'));
```

Reply:
Oh cool, that's much nicer, thank you! (/me reads up on the [difference between getBy, findBy, queryBy](https://levelup.gitconnected.com/the-difference-between-get-find-query-react-testing-library-bcd996ba3baa))

cf [5a1de30](https://github.com/openedx/frontend-app-course-authoring/pull/787/commits/5a1de307d518803fbef3d8e2d2aa8a1fe186d097)